2022/24xxx/CVE-2022-24387.json

```diff
@@ -1,18 +1,100 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'CWE-434 Unrestricted Upload of File with Dangerous Type'}]}]}